src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java

```diff
@@ -663,10 +663,18 @@ public void onDeath(DamageSource cause)
                     tavern.setNoVisitorTime(world.getRandom().nextInt(5000) + 30000);
                 }
 
+                final String deathLocation = "{".concat(String.valueOf(getCitizenData().getLastPosition().getX()))
```

Comment:
I think we have a utility method somewhere for this.

Reply:
Any idea where to look

Reply:
BlockPosUtil.getString()